extension/app/ts/background/simulationModeHanders.ts

```diff
@@ -269,11 +269,14 @@ export async function getTransactionCount(simulator: Simulator, port: browser.ru
 	})
 }
 
-export async function getSimulatedUnsignedTransactions(simulator: Simulator, port: browser.runtime.Port, request: InterceptedRequest) {
+export async function getSimulationResults(simulator: Simulator, port: browser.runtime.Port, request: InterceptedRequest) {
 	return postMessageIfStillConnected(port, {
 		interceptorApproved: true,
 		requestId: request.requestId,
 		options: request.options,
-		result: EthereumUnsignedTransactionArray.serialize(simulator.simulationModeNode.getSimulatedUnsignedTransactions())
+		result: {
+			version: '1.0.0',
+    		result: GetSimulationResults.serialize(simulator.simulationModeNode.getSimulationResults())
```

Comment:
```suggestion
    			result: GetSimulationResults.serialize(simulator.simulationModeNode.getSimulationResults())
```

Reply:
Consider: `payload` rather than `result`, as this includes not only results of simulation but also transactions in the simulation.